tiny-ab.js

```diff
@@ -0,0 +1,41 @@
+const tinyABinit = () => {
+  const contentOriginal = document.getElementsByClassName('ab-test-original');
+  const contentAlt = document.getElementsByClassName('ab-test-alternative');
+  const urlParams = new URLSearchParams(window.location.search);
+  const testCriteria = (contentAlt.length > 0) && (urlParams.has('lti_context_id'));
+
+  if (testCriteria) {
+    const id = urlParams.get('lti_context_id');
+    addCSS();
+    runTinyAB(contentOriginal, contentAlt, id);
+  }
+}
+
+const addCSS = () => {
+  const style = document.createElement('style');
+  style.innerHTML = `
+    .ab-test-original+.ab-test-alternative { display: none; }
```

Comment:
Why are we doing the {display: none} if we're potentially removing the element a second later anyway?